section2/exercises/comparisons.js

```diff
@@ -110,22 +130,27 @@ var age = 1;
 
 // YOU DO:
 // Determine if the dog loves to play and loves treats
-
+console.log(lovesToPlay && lovesTreats);
 
 // Determine if the dog loves to play and loves the dog park
-
+console.log(lovesToPlay && lovesDogPark);
 
 // Determine if the dog loves to play or loves the dog park
-
+console.log(lovesToPlay || lovesDogPark);
 
 // Determine if the dog loves to play and is a puppy
+console.log(lovesToPlay && age);
```

Comment:
Yeah, to your point below, we could instead write:

```
console.log(lovesToPlay && age > 1);
```

To get a boolean in the output